components/src/util/logger.js

```diff
@@ -0,0 +1,22 @@
+const isLogging = process.env.LOGGING;
+
+export default logger();
+
+function logger () {
+  return isLogging ? devLogger() : prodLogger();
+}
+
+function prodLogger () {
+  return {
+    log: noop,
+    info: noop,
+    error: noop,
+    warn: noop
+  };
+}
+
+function devLogger () {
+  return console;
```

Comment:
kk
let's have it in a separate PR, this logger is just reused from previous projects